dmoj/executors/mixins.py

```diff
@@ -74,10 +114,24 @@ def get_security(self, launch_kwargs=None):
         return self._add_syscalls(sec)
 
     def get_fs(self):
-        name = self.get_executor_name()
-        fs = BASE_FILESYSTEM + self.fs + env.get('extra_fs', {}).get(name, []) + [re.escape(self._dir)]
+        extra_fs = self._load_extra_fs()
+
```

Comment:
Nit: no extra newline necessary here. Honestly, the assignment to `fs` isn't, either.